src/library.js

```diff
@@ -50,7 +50,13 @@ LibraryManager.library = {
   // utime.h
   // ==========================================================================
 
-  utime__deps: ['$FS', '$setErrNo'],
+  $handleFSError__deps: ['$setErrNo'],
+  $handleFSError: function(e) {
```

Comment:
The name here is somewhat odd - `handleFSError` does not seem to handle an error from the filesystem, but it instead raises one? Would `raiseFSError` be better?

Reply:
I didn't come up with the name, I just moved this from being inside the `FS` object to outside of it.   

Happy to come up with a different name.

Its kind of the opposite of raising though.. its designed take an error object, which means it runs mostly *inside* of the exception handing that.  If the caught error is a filesytem error is handles it by setting errno... othewise is re-raises.   Don't you think that sounds like an error handler?